frontend_lib/src/component/Input/BtnSwitch/BtnSwitch.styl

```diff
@@ -1,5 +1,21 @@
 @import '../../../css/Variable.styl'
 
+.smallSize
+  & .switch
+    width 30px
+    height 17px
+  & .slider.round
+    border-radius 17px
+    &:before
+      height 13px
+      width 13px
+      left 2px
+      bottom 2px
+  & input:checked + .slider:before
+    -webkit-transform translateX(13px)
+    -ms-transform translateX(13px)
```

Comment:
```suggestion
```

We don't need those prefix anymore. Browsers have supported the prefixless rule for years.